pkgs/servers/pulseaudio/default.nix

```diff
@@ -9,7 +9,7 @@
 
 , x11Support ? false
 
-, useSystemd ? stdenv.isLinux
+, useSystemd ? (stdenv.isLinux && (!stdenv.hostPlatform == stdenv.buildPlatform || libOnly))
```

Comment:
I think this expression would be clearer if the `!` were outside the bracket (assuming that will have the same meaning - evidently this is not clear to me at least)